packages/itwin/grouping-mapping-widget/src/common/utils.ts

```diff
@@ -95,3 +96,16 @@ export const enableExperimentalFeatures = async (iModelConnection: IModelConnect
   const reader = iModelConnection.createQueryReader("PRAGMA experimental_features_enabled=true");
   await reader.step();
 };
+
+export const getLocalizedStringPresentation = (input: string): string => {
+  // https://www.itwinjs.org/presentation/advanced/localization/
+  // Check if the string starts and ends with "@"
+  if (input.startsWith("@") && input.endsWith("@")) {
+    // Remove the leading and trailing "@" from the string
+    const trimmedString = input.substring(1, input.length - 1);
+
+    return IModelApp.localization.getLocalizedString(trimmedString);
```

Comment:
I wonder if this adds a requirement to initialize the localization before rendering the Grouping & Mapping widget. I remember that Reports config widget has an initialization step because of localization. Is it the same here?

Reply:
Yes, the viewer itself already does the initialization. 
https://github.com/iTwin/viewer/blob/a58000781f538ac468368e875072d587a415186d/packages/modules/viewer-react/src/services/BaseInitializer.ts#L196